repl.js

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Copyright 2017 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+/*
+ * Usage: `node repl.js` and then `PrincipalsAPI.createUser(...);` e.g.
+ */
+
+let repl = require("repl").start({}),
+    promisify = require("repl-promised").promisify;
```

Comment:
I'd prefer these as 
```javascript
const repl = require('repl').start({});
const promisify = require('repl-promised').promisify;
```

Additionally, shouldn't repl-promised be added to package.json?

Reply:
Fixed!